english/marshmallow.md

```diff
@@ -385,3 +385,7 @@ Do you:
 [Talk to Finn](talk-to-finn/finn.md)
 
 [Play Guitar](guitar/guitar.md)
+
+[Rinse the marshmallow under the tap then eat it](dysentry/dysentry.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.